wagtailio/settings/base.py

```diff
@@ -123,7 +123,6 @@
 LANGUAGE_CODE = "en-gb"
 TIME_ZONE = "UTC"
 USE_I18N = True
-USE_L10N = True
```

Comment:
I think this is still technically needed. Imagine we don’t need this in [Django 4.0 and up](https://docs.djangoproject.com/en/4.1/ref/settings/#use-l10n), but right now the PR doesn’t actually update to Django 4.1?